sources/utils.py

```diff
@@ -377,3 +377,18 @@ def split_list(l, nb_partitions):
     """
     return [l[i:(i+len(l)/nb_partitions)]
             for i in range(0, len(l), int(len(l)/nb_partitions))]
+
+def list_to_str(l):
```

Comment:
I prefer having a variable name with more than one letter here. I like `seq` for *sequence*. It can be a list, a tuple, etc. It's a collection of strings. Or another name > 1 letter :)

Reply:
Done.